### PR TITLE
[JBPM-9423] Duplicate container-info entries in kie-server-router.jso…

### PR DESCRIPTION
**Pull request by elguardian**
Jira: https://issues.redhat.com/browse/JBPM-9423

### CODE REVIEW

File: kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/ConfigurationTest.java [hunk 1]
```diff
@@ -25,6 +26,56 @@
...
+        configuration.addServerHost(serverId, serverUrl);
+        configuration.addContainerInfo(containerInfo);
+
+        assertEquals(2, configuration.getHostsPerContainer().size());
+        assertEquals(1, configuration.getHostsPerServer().size());
+
+        assertEquals(3, configuration.getHostsPerContainer().get("itorders_1.01").size());
+        assertEquals(1, configuration.getHostsPerContainer().get("itorders_1.0.1").size());
+        assertEquals(3, configuration.getHostsPerServer().get("default-kieServer").size());
+        assertEquals(3, configuration.getHostsPerServer().get("default-kieServer").size());
```
Comment (gmunozfe) [inline kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/ConfigurationTest.java:77]: This line is duplicated

**Review by gmunozfe — CHANGES_REQUESTED** [review]
Test failures related with the PR:
https://github.com/kiegroup/droolsjbpm-integration/blob/2d3b1c96e52117b7f7179b30c4e38270d7a6a900/kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/ConfigurationTest.java#L152-L153
A different alias is needed to fix that test

File: kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/ConfigurationTest.java (hunk 1 above)
Comment (elguardian) [reply kie-server-parent/kie-server-router/kie-server-router-proxy/src/test/java/org/kie/server/router/ConfigurationTest.java]: done

**Review by gmunozfe — APPROVED** [review]
Just a single comment, if you want to handle it by refactoring and extracting the configuration "add..." invocations.
Otherwise, looks good to me, fantastic work @elguardian !

**Pull request merged**